main.py

```diff
@@ -35,13 +36,19 @@ def cancel(bot, update):
 msg = range(1)
 
 def spam_callback(bot, update):
-    insert_message(update.message, update.message.from_user.id, "spam")
+    raw_user_id = update.message.from_user.id
+    hash_user_id = sha256(str(raw_user_id).encode()).hexdigest()
```

Comment:
Can you make it a function named "encode_id" and call it here instead?
```suggestion
    hash_user_id = encode_id(raw_user_id)
```

Reply:
Done